keras/src/dtype_policies/dtype_policy.py

```diff
@@ -202,6 +202,10 @@ def __repr__(self):
         return f'<FloatDTypePolicy "{self._name}">'
 
 
+GLOBAL_DEFAULT_PLACEHOLDER = "global_default"
```

Comment:
> Why use this string as the initial value, instead of e.g. None?

Currently, `DTypePolicy` and its subclasses rely on string value for parsing.
It is not clear for me how we can pass `None` in combination with the quantization mode.

Should we refactor `QuantizedDTypePolicy` to support a signature for both the quantization mode and the source dtype policy?

Ex:
```python
policy = QuantizedDTypePolicy(mode="int8", source_dtype_policy="mixed_bfloat16")
```

Reply:
> Currently, DTypePolicy and its subclasses rely on string value for parsing.
It is not clear for me how we can pass None in combination with the quantization mode.

We could just modify `DTypePolicy` to support `None`, meaning "default".

> Should we refactor QuantizedDTypePolicy to support a signature for both the quantization mode and the source dtype policy?

Yes, that's a great idea!